controllers/main.js

```diff
@@ -10,15 +10,28 @@ const {
   resultsStyle,
   detailedResultsStyle,
 } = require("../public/javascripts/extraStyles.js");
+const remapDifficulty = {
+  1: "Very Easy",
+  2: "Easy",
+  3: "Medium",
+  4: "Hard",
+  5: "Very Hard",
+};
 
 module.exports.playOrContinue = async (req, res) => {
-  const user = req.user.username;
-  const lastSessionData = await fetchPlaySessionData(user);
-  if (
-    lastSessionData.sessionData.length &&
-    lastSessionData.sessionData.length !== lastSessionData.length
-  ) {
-    res.redirect("/play/continue");
+  if (req.user?._id) {
+    const user = req.user._id;
```

Comment:
I'm not sure why you declared a constant here and assigned a value that will be used just once. Why not just pass the value as is without storing it in a constant?

`const lastSessionData = await fetchPlaySessionData(req.user._id);`

Reply:
I guess it became a bit of a habit to declare constants like this right away regardless of how necessary that is.